<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QTabWidget" name="renameMode_tabWidget" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>0</y>
      <width>401</width>
      <height>431</height>
     </rect>
    </property>
    <property name="palette" >
     <palette>
      <active>
       <colorrole role="Button" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>65</red>
          <green>69</green>
          <blue>88</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>65</red>
          <green>69</green>
          <blue>88</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>65</red>
          <green>69</green>
          <blue>88</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font" >
     <font>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="autoFillBackground" >
     <bool>false</bool>
    </property>
    <property name="currentIndex" >
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_2" >
     <attribute name="title" >
      <string>Filter</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>240</y>
        <width>191</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title" >
       <string>Selection:</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QListWidget" name="selection_listWidget" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>15</y>
         <width>191</width>
         <height>141</height>
        </rect>
       </property>
       <property name="editTriggers" >
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode" >
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="selectionRectVisible" >
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3" >
      <property name="geometry" >
       <rect>
        <x>200</x>
        <y>240</y>
        <width>191</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title" >
       <string>Preview Results:</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QListWidget" name="previewResults_listWidget" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>15</y>
         <width>191</width>
         <height>141</height>
        </rect>
       </property>
       <property name="selectionMode" >
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="searchAndReplace_groupBox" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>10</y>
        <width>381</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title" >
       <string>Search and Replace</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="flat" >
       <bool>false</bool>
      </property>
      <property name="checkable" >
       <bool>false</bool>
      </property>
      <widget class="QLabel" name="label_11" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>70</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text" >
        <string>Scope:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>47</y>
         <width>46</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text" >
        <string>Case:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>20</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text" >
        <string>Search For:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="searchScope_comboBox" >
       <property name="geometry" >
        <rect>
         <x>150</x>
         <y>70</y>
         <width>211</width>
         <height>16</height>
        </rect>
       </property>
       <item>
        <property name="text" >
         <string>Only those with Occurrence</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Only those Starting with</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Only those Ending with</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="searchCaseSensitive_comboBox" >
       <property name="geometry" >
        <rect>
         <x>60</x>
         <y>50</y>
         <width>291</width>
         <height>16</height>
        </rect>
       </property>
       <item>
        <property name="text" >
         <string>case sensitive</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>ignore case</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="search_lineEdit" >
       <property name="geometry" >
        <rect>
         <x>130</x>
         <y>20</y>
         <width>231</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="scopeKeepPrune_comboBox" >
       <property name="geometry" >
        <rect>
         <x>60</x>
         <y>70</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <item>
        <property name="text" >
         <string>keep</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>prune</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_12" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>120</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text" >
        <string>Rule1:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="rule1_comboBoxKeepPrune_comboBox" >
       <property name="geometry" >
        <rect>
         <x>50</x>
         <y>120</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <item>
        <property name="text" >
         <string>keep</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>prune</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="rule1_comboBox" >
       <property name="geometry" >
        <rect>
         <x>140</x>
         <y>120</y>
         <width>184</width>
         <height>16</height>
        </rect>
       </property>
       <item>
        <property name="text" >
         <string/>
        </property>
       </item>
       <item>
        <property name="text" >
         <string/>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="rule2_comboBox" >
       <property name="geometry" >
        <rect>
         <x>140</x>
         <y>140</y>
         <width>184</width>
         <height>16</height>
        </rect>
       </property>
       <item>
        <property name="text" >
         <string/>
        </property>
       </item>
       <item>
        <property name="text" >
         <string/>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_13" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>140</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text" >
        <string>Rule2:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="rule2_comboBoxKeepPrune_comboBox" >
       <property name="geometry" >
        <rect>
         <x>50</x>
         <y>140</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <item>
        <property name="text" >
         <string>keep</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>prune</string>
        </property>
       </item>
      </widget>
      <widget class="QFrame" name="frame" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>90</y>
         <width>351</width>
         <height>5</height>
        </rect>
       </property>
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QCheckBox" name="rule1_considerShapecheckBox" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="geometry" >
        <rect>
         <x>330</x>
         <y>120</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="checked" >
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="rule2_considerShapecheckBox" >
       <property name="geometry" >
        <rect>
         <x>330</x>
         <y>140</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="checked" >
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_14" >
       <property name="geometry" >
        <rect>
         <x>330</x>
         <y>100</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font" >
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>Consider
Shapes:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15" >
       <property name="geometry" >
        <rect>
         <x>140</x>
         <y>100</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font" >
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>Node Type:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16" >
       <property name="geometry" >
        <rect>
         <x>50</x>
         <y>100</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font" >
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>filter operation:</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="filter_button" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>210</y>
        <width>361</width>
        <height>21</height>
       </rect>
      </property>
      <property name="palette" >
       <palette>
        <active>
         <colorrole role="WindowText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>170</red>
            <green>85</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Light" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>128</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Midlight" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>212</red>
            <green>106</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Dark" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>85</red>
            <green>42</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Mid" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>113</red>
            <green>56</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="BrightText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>170</red>
            <green>85</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Shadow" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="AlternateBase" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>212</red>
            <green>170</green>
            <blue>127</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ToolTipBase" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>220</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ToolTipText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>170</red>
            <green>85</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Light" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>128</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Midlight" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>212</red>
            <green>106</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Dark" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>85</red>
            <green>42</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Mid" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>113</red>
            <green>56</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="BrightText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>170</red>
            <green>85</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Shadow" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="AlternateBase" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>212</red>
            <green>170</green>
            <blue>127</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ToolTipBase" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>220</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ToolTipText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>85</red>
            <green>42</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>170</red>
            <green>85</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Light" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>128</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Midlight" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>212</red>
            <green>106</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Dark" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>85</red>
            <green>42</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Mid" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>113</red>
            <green>56</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>85</red>
            <green>42</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="BrightText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>85</red>
            <green>42</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>170</red>
            <green>85</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>170</red>
            <green>85</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Shadow" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="AlternateBase" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>170</red>
            <green>85</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ToolTipBase" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>220</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ToolTipText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="text" >
       <string>Filter</string>
      </property>
     </widget>
     <zorder>filter_button</zorder>
     <zorder>groupBox_4</zorder>
     <zorder>groupBox_3</zorder>
     <zorder>searchAndReplace_groupBox</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
